### DISCUSSION TITLE
Pod ephemeral-storage

### DISCUSSION THREAD

**Discussion by patricio-dorantes** [prometheus-operator/kube-prometheus#953] [Q&A]
Pod ephemeral storage reporting

Hi! 
I have been looking for information on k8s ephemeral storage limit enforce and monitor. I have installed Prometheus operator and I manage only to get the limits I have set on the pod but not the usage. I have found on the net [this k8s issue](https://github.com/kubernetes/kubernetes/issues/69507) rejecting the kubelet ephemeral storage info, and [https://github.com/kubernetes/kubernetes/issues/78865](this k8s issue) that made me want to know if limit.ephemeral-storage was being reported on my k8s cluster.

Q: are this independent issues (available metrics and limits not being enforced)? 
Q: Is it really a cardinality problem to have pod ephemeral storage usage? 

Thank you guys in advance.

**Comment by brancz** [comment]
I personally don't think there is a cardinality problem. Sure it's roughly O(pods) but this is very useful data when there is a problem.
